src/nft/utils/currency.ts

```diff
@@ -41,24 +41,26 @@ export const numberToWei = (amount: number) => {
   return parseEther(amount.toString())
 }
 
-export const ethNumberStandardFormatter = (amount: string | number | undefined, includeDollarSign = false): string => {
+export const ethNumberStandardFormatter = (
+  amount: string | number | undefined,
+  includeDollarSign = false,
+  removeZeroes = false
+): string => {
   if (!amount) return '-'
 
   const amountInDecimals = parseFloat(amount.toString())
   const conditionalDollarSign = includeDollarSign ? '$' : ''
 
   if (amountInDecimals < 0.0001) return `< ${conditionalDollarSign}0.00001`
   if (amountInDecimals < 1) return `${conditionalDollarSign}${amountInDecimals.toFixed(3)}`
-  return (
-    conditionalDollarSign +
-    amountInDecimals
-      .toFixed(2)
-      .toString()
-      .replace(/\B(?=(\d{3})+(?!\d))/g, ',')
-  )
+  const formattedPrice = amountInDecimals
+    .toFixed(2)
+    .toString()
+    .replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  return conditionalDollarSign + (removeZeroes ? formattedPrice.replace(/.00/g, '') : formattedPrice)
```

Comment:
Do we also want to remove any trailing zeros not just if there's double behind the decimal? For numbers like `3.40` for example

Reply:
design agreed - updated across app besides bag